base/service/src/main/java/org/eclipse/ditto/base/service/devops/LogbackLoggingFacade.java

```diff
@@ -73,17 +73,24 @@ public List<LoggerConfig> getLoggerConfig(@Nonnull final Iterable<String> logger
                 .map(logger -> (Logger) LoggerFactory.getLogger(logger))
                 .toList();
 
-        return loggerConfigsFor(loggerList);
+        return loggerConfigsFor(loggerList, true);
```

Comment:
Is there a reason this is statically true?
Shouldn't this be either controlled via a parameter or set to false to resemble the prior behaviour?

Reply:
Changed